src/main/java/com/ent/staySafe/staySafeController.java

```diff
@@ -7,7 +7,7 @@
 
 @Controller
 
-public class staySafeController {
+public class StaySafeController {
 	@RequestMapping("/staySafe")
```

Comment:
Oracle Java Class naming conventions, "Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML)."
I capitalized this Class name to adhere to these conventions

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html